.storybook/preview.tsx

```diff
@@ -14,38 +14,38 @@ import results from "../.jest-test-results.json";
 // Custom viewport options
 const customViewports = {
   sm: {
-    name: "sm (320px)",
+    name: "sm - small mobile (less than 480)",
     styles: {
-      width: "320px",
-      height: "568px",
+      width: "390px",
```

Comment:
How did you get these values?

Reply:
These values align with the Figma art board sizes. And the art board sizes were selected because they were good representative sizes within the breakpoint ranges.